Visma.net/Models/Project.cs

```diff
@@ -31,12 +37,20 @@ public Project()
         public string description { get { return Get<string>(); } set { Set(value); } }
         public DateTime startDate { get { return Get<DateTime>(); } set { Set(value); } }
         public DateTime endDate { get { return Get<DateTime>(); } set { Set(value); } }
-        [JsonProperty]
-        public ProjectManager projectManager { get { return Get<ProjectManager>(); } private set { Set(value); } }
+
+        public ProjectManager projectManager {
```

Comment:
Why are you setting employeeNumber but getting projectManger (sic)? Is this an error?

Reply:
When getting project from visma, you gets meta info of the project manager
![image](https://user-images.githubusercontent.com/24973859/96690889-8735e880-1384-11eb-8b7a-50c47bf51cb5.png)
but when posting project project is just an emp id.
![image](https://user-images.githubusercontent.com/24973859/96691051-b8aeb400-1384-11eb-8eed-869cc6bc792a.png)

Could maybe be solved an other way?

Reply:
Maybe ProjectManager should be a IBecomeDto with `ToDto() => new DtoValue(employeeNumber)`?

Reply:
I Changed the ProjectManager class now.